lib/base/RequestClient.js

```diff
@@ -53,6 +54,10 @@ RequestClient.prototype.request = function(opts) {
     forever: opts.forever === false ? false : true,
   };
 
+  if (process.env.TWILIO_CA_BUNDLE !== undefined) {
```

Comment:
Could you also add a block to the README about this new env var and the existing ones that are supported?

```
TWILIO_ACCOUNT_SID ...
TWILIO_AUTH_TOKEN ...
TWILIO_CA_BUNDLE ...
```